### ISSUE TITLE
Please Make Mention of BrowserLink's Use for Hot Reload

### ISSUE THREAD

**Issue by Mike-E-angelo** [dotnet/AspNetCore.Docs#23053]
Greetings.  I recently installed VS2022 and noticed BrowserLink was being enabled in my solution and doing all sorts of unexpected actions/requests in the form of output debug messages in my Blazor server-side application.

Thinking this was a bug, [I reported it to developercommunity](https://developercommunity.visualstudio.com/t/browserlink-attempts-to-connect-even-though-it-is/1501863).

This page was one of the referenced pages I found in my search to find out what the heck was going on.  It would be nice to see something (note/important/etc) to outline its new affiliation with Hot Reload so others (who pay attention 😁) will know what's happening in their new environment.

Thank you for any consideration.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1fcc1fa9-004b-0be3-6148-e8c075e5b2bc
* Version Independent ID: 78a1947e-dea7-65b4-2376-05c7795e4bf9
* Content: [Browser Link in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/client-side/using-browserlink?view=aspnetcore-6.0)
* Content Source: [aspnetcore/client-side/using-browserlink.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/client-side/using-browserlink.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-clientside**
* GitHub Login: @ncarandini
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
@guardrex  is this more problematic with Blazor?

**Comment by guardrex** [comment]
It's not covered yet for Blazor due to engineering churn. However, it should be addressed fairly soon ... next couple of weeks. I believe that you can close this issue because it's already tracked by https://github.com/dotnet/AspNetCore.Docs/issues/22440.

**Comment by Mike-E-angelo** [comment]
Indeed, there is absolutely no documentation out there around this so it was a bit of a struggle, hence the developercommunity report.  I am OK with closing this issue if it's captured in a larger story.  My concern was for others who might be running into this as I have figured things out on my end after all the pain. :)

Thank you for all your great work over there. 👍

**Comment by guardrex** [comment]
I think the blog posts discuss it *somewhere*. Search through them to see. I keep a list of handy links at ...

https://github.com/dotnet/AspNetCore.Docs/issues/22045

... scroll down to the list of them.

We don't document things like that **_tooooo early_** because engineering churn can easily break preview docs costing a lot of time ⌚ and money 💰. Indeed, they even **drop** features sometimes, and we have had to delete new coverage that we generated when that happens. We try to get everything documented by the late preview and RC stages for GA. Until then, devs should use the blog posts and the engineering issues and PRs to use preview technology.
